templates/optaweb/rhdm71-optaweb-employee-rostering-externaldb.yaml

```diff
@@ -0,0 +1,380 @@
+  metadata:
+    name: "${APPLICATION_NAME}-optaweb-employee-rostering"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-optaweb-employee-rostering"
+    annotations:
+      template.alpha.openshift.io/wait-for-ready: "true"
+  spec:
+    strategy:
+      type: Recreate
+    triggers:
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+        - "${APPLICATION_NAME}-optaweb-employee-rostering"
+        from:
+          kind: ImageStreamTag
+          namespace: "${IMAGE_STREAM_NAMESPACE}"
+          name: "${OPTAWEB_IMAGE_STREAM_NAME}:${IMAGE_STREAM_TAG}"
+    - type: ConfigChange
+    replicas: 1
+    selector:
+      deploymentConfig: "${APPLICATION_NAME}-optaweb-employee-rostering"
+    template:
+      metadata:
+        name: "${APPLICATION_NAME}-optaweb-employee-rostering"
+        labels:
+          deploymentConfig: "${APPLICATION_NAME}-optaweb-employee-rostering"
+          application: "${APPLICATION_NAME}"
+          service: "${APPLICATION_NAME}-optaweb-employee-rostering"
+      spec:
+        serviceAccountName: "${APPLICATION_NAME}-optaweb-employee-rostering"
+        terminationGracePeriodSeconds: 60
+        containers:
+        - name: "${APPLICATION_NAME}-optaweb-employee-rostering"
+          image: "${OPTAWEB_IMAGE_STREAM_NAME}"
+          imagePullPolicy: Always
+          resources:
+            limits:
+              memory: "${OPTAWEB_MEMORY_LIMIT}"
+          volumeMounts:
+          - name: optaweb-employee-rostering-keystore-volume
+            mountPath: "/etc/optaweb-employee-rostering-secret-volume"
+            readOnly: true
+          livenessProbe:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-c"
+              - "curl --fail --silent -u '${KIE_ADMIN_USER}:${KIE_ADMIN_PWD}' http://localhost:8080/swagger/index.html"
+            initialDelaySeconds: 180
+            timeoutSeconds: 2
+            periodSeconds: 15
+            failureThreshold: 3
+          readinessProbe:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-c"
+              - "curl --fail --silent -u '${KIE_ADMIN_USER}:${KIE_ADMIN_PWD}' http://localhost:8080/swagger/index.html"
+            initialDelaySeconds: 60
+            timeoutSeconds: 2
+            periodSeconds: 30
+            failureThreshold: 6
+          ports:
+          - name: jolokia
+            containerPort: 8778
+            protocol: TCP
+          - name: http
+            containerPort: 8080
+            protocol: TCP
+          - name: https
+            containerPort: 8443
+            protocol: TCP
+          - name: ping
+            containerPort: 8888
+            protocol: TCP
+          env:
+          - name: KIE_ADMIN_USER
+            value: "${OPTAWEB_ADMIN_USER}"
+          - name: KIE_ADMIN_PWD
+            value: "${OPTAWEB_ADMIN_PWD}"
+          - name: DATASOURCES
```

Comment:
Would also be good to add the DRIVERS env?
it will allow user to install drivers as modules and create the proper configuration for it on standalone.xml

Reply:
@spolti I modeled this template after the RHPAM externaldb one, which does not include the DRIVERS env... If it should be done, perhaps we should do it separately, for both?

Reply:
Ok, we could add as part of https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/142

Reply:
Agreed, but obviously it would be in a different PR since it's DM not PAM.